### PR TITLE
[Hotfix] Removed credman dependencies

### PR DESCRIPTION
**Pull request by somalaya**
Updated common version which has the changes to remove credman dependencies
Also made robolectric sdk upgrade to run the UTs in monthly pipeline smoothly

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,5 +1,9 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 5.0.1
+----------
+- [PATCH] Update robolectricVersion 4.11.1 (#2015)
```
Comment (mohitc1) [inline changelog:5]: nit: changelog update common@17.0.1

**Review by mohitc1 — APPROVED** [review]

File: changelog (hunk 1 above)
Comment (somalaya) [reply changelog:5]: done

File: gradle/versions.gradle [hunk 2]
```diff
@@ -43,7 +43,7 @@ ext {
     powerMockVersion = "2.0.9"
     runnerVersion = "1.2.0"
     rulesVersion = "1.2.0"
-    robolectricVersion = "4.9.2"
+    robolectricVersion = "4.11.1"
```
Comment (fadidurah) [inline gradle/versions.gradle:46]: Do we need this?

Comment (somalaya) [reply gradle/versions.gradle:46]: made robolectric sdk upgrade to run the UTs in monthly pipeline smoothly

**Review by fadidurah — APPROVED** [review]

**Pull request merged**